dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -73,15 +73,15 @@ ENV BEAT_SETUID_AS={{ .user }}
 {{- if contains .from "ubi-minimal" }}
 RUN for iter in {1..10}; do \
         microdnf update -y && \
-        microdnf install -y tar gzip findutils shadow-utils ca-certificates gawk libcap xz && \
+        microdnf install -y tar gzip findutils shadow-utils ca-certificates gawk libcap xz systemd && \
```

Comment:
How much bigger does this make the images? 

Also, we don't want it in every container, it would be in elastic-agent, elastic-agent-complete but not elastic-agent-slim. This is really dependent on how much the size increase is as we do not want them getting bigger since they are already huge.

Reply:
The downside of using the elastic-agent-complete image is that people who don't use synthetics but want to read node logs on K8S now have to deploy a bunch of browser dependencies and such.

Perhaps this would be best addressed with an elastic-agent-journald container. We already have a lot of containers though so I'm not sure this clarifies anything with respect to which one someone should use.